libobs/obs-encoder.h

```diff
@@ -102,6 +103,23 @@ struct encoder_frame {
 	int64_t pts;
 };
 
+/** Encoder region of interest */
+struct region_of_interest {
```

Comment:
In my opinion, it would be better to have a prefix of `obs_` such as `obs_encoder_roi` or `obs_region_of_interest`.

Reply:
We use fairly generic names like `encoder_packet` in the encoder interface so I figured this is probably fine. We seem to mostly use `obs_` prefixes for structs that are opaque types but I don't know if this is a rule.

Reply:
There weren't any particular rules. When I used encoder_packet it was probably because it sounded better than obs_encoder_packet and I felt like it probably wasn't going to clash with anything. I probably should have tried to maintain some level of consistency. I'd say we should *probably* try to lean towards consistent behavior so maybe norihiro's suggestion is for the best or something.

Reply:
+1 for `obs_encoder_roi`, and also moving towards a naming syntax including a prefix of `obs_[category]_`